soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -144,6 +144,7 @@ static ALWAYS_INLINE void _restore_core_context(void)
 	XTENSA_WSR("EXCSAVE2", core_desc[core_id].excsave2);
 	XTENSA_WSR("EXCSAVE3", core_desc[core_id].excsave3);
 	XTENSA_WUR("THREADPTR", core_desc[core_id].thread_ptr);
+	__asm__ volatile("wsr.intenable %0" : : "r"(0x00000000));
```

Comment:
This step is redundant; the register is cleared in the `cpu_early_init` procedure.

Reply:
yep. Thanks for pointing it.